packages/ra-ui-materialui/src/detail/Show.spec.tsx

```diff
@@ -117,7 +117,7 @@ describe('<Show />', () => {
 
     it('should display a default title based on resource and id', async () => {
         render(<Basic />);
-        await screen.findByText('Book #1');
+        await screen.findByText('Book War and Peace');
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** The title no longer describes what the test does. We should rename the test IMO‏